README.md

```diff
@@ -13,8 +13,8 @@ For more information about this actuator and related repositories, see
 
 | CAPM3 version | Cluster API version | CAPM3 Release |
 |---------------|---------------------|---------------|
-| v1alpha3      | v1alpha3            | v0.3.X        |
```

Comment:
I think it would be nice to leave the old version in this table for a while so it is possible to easily see what versions are compatible. If we want to point out that v0.3.X is deprecated and no longer supported we could add ~~strike through~~ and/or `deprecated` next to it.

Reply:
Good point. On it.

Reply:
Done